proto/data.go

```diff
@@ -315,6 +315,16 @@ func (v *Value) GetInteger() (int64, error) {
 	return int64(u), nil
 }
 
+// SetMessage encodes the specified message into the bytes field of the receiver.
+func (v *Value) SetMessage(msg gogoproto.Message) error {
```

Comment:
`s/SetMessage/SetProto/g`. We use the term `Proto` elsewhere in method names like this. For example, `client/DB.GetProto`.